tests/integration/classification/multiclass/test_workflow.py

```diff
@@ -31,11 +28,26 @@
 from tests.integration.helper import with_test_prefix
 
 
-@pytest.mark.parametrize("inference_type", [ScoredClassificationLabel, InferenceLabel])
-def test__multiclass_classification__smoke(
-    inference_type: Union[Type[ScoredClassificationLabel], Type[InferenceLabel]],
-) -> None:
-    name = with_test_prefix(f"{__file__} test__multiclass_classification__smoke ({inference_type.__name__})")
+def test__multiclass_classification__smoke__scored__classification_label() -> None:
+    name = with_test_prefix(
+        f"{__file__} test__multiclass_classification__smoke (ScoredClassificationLabel) {str(uuid.uuid4())}",
```

Comment:
`with_test_prefix` includes a session-wide random string already, is the additional uuid needed for correctness?

Reply:
In this case, no - just an extra layer to ensure the test cases are unique.